src/widgets/tree/rc-tree/TreeNode.js

```diff
@@ -436,16 +446,39 @@ class TreeNode extends React.Component {
     return getTreeThemeDefaultConfig(size, target)[state];
   }
 
+  getSelectedParentTextThemeProps = (target, params) => {
+    const themeProps = this.props.getPartOfThemeProps(target, params);
+    const { themeConfig = {} } = themeProps;
+    const { normal = {} } = themeConfig;
+    const { themeConfig: parentTexThemeConfig = {} } = this.props.getPartOfThemeProps(
+      'SelectedParentText'
+    );
+    const { normal: TextExpanded = {} } = this.props.getPartOfThemeProps('TextExpanded');
+    themeConfig.normal = deepMerge(
+      deepMerge(
+        this.getDefaultTitleWrapTheme('normal', params.props, target),
+        normal,
+        parentTexThemeConfig.normal
+      ),
+      TextExpanded
+    );
+    return themeProps;
+  };
+
   getTitleWrapThemeProps(defaultName: string, selectedName: string, params: Object = {}): Object {
     const { parentIsHighlight } = this.props;
-
-    return this.isChecked() || parentIsHighlight
-      ? this.getSelectedTitleWrapThemeProps(selectedName, {
-          props: params,
-        })
-      : this.getDefaultTitleWrapThemeProps(defaultName, {
-          props: params,
-        });
+    if (this.isChecked()) {
+      return this.getSelectedTitleWrapThemeProps(selectedName, {
+        props: params,
+      });
+    } else if (parentIsHighlight) {
+      return this.getSelectedParentTextThemeProps(selectedName, {
+        props: params,
+      });
+    }
+    return this.getDefaultTitleWrapThemeProps(defaultName, {
+      props: params,
+    });
```

Comment:
const xxxxx= this.isChecked() || parentIsHighlight ? selectedName:defaultName;
return  return this.getDefaultTitleWrapThemeProps( xxxxx, {
      props: params,
    })